### ISSUE TITLE
tar noises transmit way too far

### ISSUE THREAD

**Issue by fluxionary** [Terumoc/terumet#34]
Not entirely sure why this is happening; max_hear_distance should default to 32, and you don't specify it, but players are hearing it over 5000 nodes away, but not at 30000 nodes away.

**Comment by CubicMelon** [comment]
Wow that is bizarre! I can add a max_hear_distance of course to try and fix it thought like you said I didn't expect that to be a problem.

**Comment by CubicMelon** [comment]
OK latest commit _should_ fix it. If there's still problems, go ahead and mention it here!

**Comment by fluxionary** [comment]
Sorry I wasn't around when you popped into the server earlier. The server was busy at that point anyway, I just now rebooted with the most current updates. The tar squish noise is still audible up to 4800-or-so blocks away from the source. There is no attenuation - it has the same volume no matter how far away I am. 

However, it only happens if the minetest client that is far from the noise source still has that area loaded in memory. If the area isn't in memory, no noises, no matter how close I am. 

Just did a little further testing - the cutoff for when I can hear the noise seems to exactly match up to when I can see the name tag of the player generating the noise (which on our server seems to be 4800 nodes).

Not sure if any of this will help solve the issue. 

This isn't a super-high-priority issue for me. If no one figures this out in the next couple days, I'll probably set aside some time to find the cause and solution at that point.

**Comment by CubicMelon** [comment]
No problem! I figured you must have been busy at the time.

Well I can imagine it must be pretty annoying for some players. If we really can't figure out how to prevent the problem I can always update it so it is without the sounds until it's fixed. (Though the tar blocks are way better with the sounds lol)

I'm wondering if it's something to do with the engine? Since as far as I know, the sound should only be audible as far as I've defined in the Sound spec.

**Comment by fluxionary** [comment]
I figured it out.
Apparently, Minetest interprets all stereo sounds as non-positional; the only way to get the engine to play a sound positionally is if it is mono. Your sounds (at least the step sounds) are all stereo.
https://dev.minetest.net/sound_play
I did enough testing to convince myself that is indeed the issue with your sounds. 
I don't think I have the energy at the moment to look up the right mplayer or ffmpeg options to strip one of the channels out of all your oggs, so I'm hoping you have an process in place already that'd make this easy.
